### PR TITLE
Fix union header

### PR DESCRIPTION
**Pull request by neithernut**
This is a fix which was previously included in the PR #235.

### CODE REVIEW

**Comment by neithernut** [comment]
Can I get an ACK?

File: src/values/union.h [hunk 1]
```diff
@@ -37,8 +47,10 @@
...
     struct ws_value_nil     nil;    //!< nil value
-    struct ws_value_bool    bool;   //!< bool value
-    struct ws_value_int     int;    //!< int value
+    struct ws_value_bool    bool_;  //!< bool value
+    struct ws_value_int     int_;   //!< int value
     struct ws_value_string  string; //!< string value
 };
 
+#endif //__WS_VALUES_UNION_H__
+
```
Comment (matthiasbeyer) [inline src/values/union.h:56]: You must close the documentation group here.

``` c
/**
 * @}
 */
```

Comment (neithernut) [inline src/values/union.h:56]: Good to know, I'll close this PR anyway because other things are missing.
I'll open another one.

**Comment by neithernut** [comment]
Close unmerged as the scope of the PR is offset from what I really want/need.

**Comment by matthiasbeyer** [comment]
Also: Why is this tagged with the bug label?

**Comment by matthiasbeyer** [comment]
huh,... okay

**Pull request closed without merging**